alacritty/src/config/window.rs

```diff
@@ -4,6 +4,8 @@ use log::error;
 use serde::{Deserialize, Deserializer};
 use serde_yaml::Value;
 
+use glutin::window::Fullscreen;
```

Comment:
```suggestion
use glutin::window::Fullscreen;
```

It's not that special here, so I think we can just add it to the block with all other dependencies when there's just one of them.